spec/lang/syntax.md

```diff
@@ -477,4 +490,13 @@ pub struct Relocation {
     /// The offset within that allocation.
     pub offset: Offset,
 }
+
+/// A vtable for a trait-type pair.
+/// This is pointed to by the trait object metadata.
+pub struct VTable {
```

Comment:
This needs to store some sort of identifier of the trait that this is a vtable for (for UB checking).